PWE.html

```diff
@@ -30,61 +32,118 @@
+    <section id="sotd"></section>
+
+        <section id="introduction">
+            <h2>Introduction</h2>
+<p>As we engage with one another, incidents can happen. Incidents can take many forms, but participants have the right to a safe and positive work environment. The goal of this incident resolution process is that participants in W3C can:</p>
+
+            <ul>
+                <li>Get help and support to safely resolve issues that negatively affect them</li>
+                <li>Correct behaviours that negatively affect other participants in W3C</li>
+                <li>Continue to safely work and contribute to the W3C after an issue has been raised</li>
+            </ul>
+                
+            <p>The Code of Conduct is the standard that all W3C participants are required to follow and is used as the basis for any resolution and disciplinary actions. In exceptional circumstances, individuals may have their W3C participation suspended or withdrawn as a result of a complaint or investigation.</p>
+                
+            <p>All situations are different and W3C participants are encouraged to use whichever method of incident resolution they feel comfortable with.</p>
+                
+            <p>Participants are encouraged to try and resolve issues themselves in the first instance. However, if participants do not feel comfortable or safe doing so, they can get help through an ombudsperson, facilitators and/or mediation.</p>
+                
+            <p>Depending on the severity of the situation, it may not be appropriate to attempt to resolve the issue informally or in the context of the group. Participants should immediately report an incident whenever they feel unsafe or threatened by the behaviour or actions of other W3C participants.</p>
+                
+            <p>Everyone involved in the resolution process is expected to:</p>
+            
+            <ul>
+                <li>Show respect for others</li>
+                <li>Work together to resolve the complaint</li>
+                <li>Maintain confidentiality</li>
+            </ul>
+                
+            <p>A disciplinary investigation can be started where one of the following apply:</p>
+                
+            <ul>
+                <li>It’s not appropriate to begin with mediation</li> 
+                <li>All other mediation or resolution attempts have not worked</li>
+            </ul>
+        </section>
+        
+<h2> Dispute Resolution Guidance</h2>
+
+<p>The goal of dispute resolution is that individual participants in W3C can:</p>
+<ul><li>get help and support to safely resolve issues that negatively affect them</li>
+<li>correct their behaviors that negatively affect other participants in W3C</li>
+        <li>continue to safely work and contribute to the W3C after an issue has been raised</li>
+</ul>

```

Comment:
The goal was already stated in the Introduction.  It does not need to be repeated here.  Better to cut the above paragraph.